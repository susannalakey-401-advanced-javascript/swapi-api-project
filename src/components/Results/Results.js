import React from 'react';
import './Results.scss';





const Results = ({ body, headers }) => {
  if (!body) {
    return null;
  }

  return (
    <>
      <div className="results">
        <pre>{JSON.stringify(headers, null, 2)}</pre>
        <pre>{JSON.stringify(body, null, 2)}</pre>
      </div>
    </>
  )
}

export default Results