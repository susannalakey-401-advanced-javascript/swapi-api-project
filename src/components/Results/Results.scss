.results {
  background-color: white;
  padding: 35px 50px;
  margin-top: 20px;
}