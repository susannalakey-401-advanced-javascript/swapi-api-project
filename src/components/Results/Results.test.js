import React from 'react'
import Results from './Results'
import { shallow } from 'enzyme'

describe('<Results />', () => {
  let tree;

  beforeEach(() => {
    const headers = {
      things: 'hello',
    }

    const body = {
      stuff: 'stuff'
    }
    tree = shallow(<Results headers={headers} body={body} />)
  })

  it('renders to the page', () => {
    expect(tree).toMatchSnapshot();
  })
})