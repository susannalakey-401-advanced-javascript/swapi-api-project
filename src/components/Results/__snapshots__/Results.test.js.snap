// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Results /> renders to the page 1`] = `
<Fragment>
  <div
    className="results"
  >
    <pre>
      {
  "things": "hello"
}
    </pre>
    <pre>
      {
  "stuff": "stuff"
}
    </pre>
  </div>
</Fragment>
`;
